src/server.ts

```diff
@@ -0,0 +1,29 @@
+import express, { Express, NextFunction, Request } from 'express'
+import { setup, serve } from 'swagger-ui-express'
+
+import cors from 'cors'
+import { json } from 'body-parser'
+import errorHandler from './lib/error-handler'
+
+import { RegisterRoutes } from './routes'
```

Comment:
is there supposed to be a `routes` folder?  If so please add it and `.gitkeep`

Reply:
thanks, never used .gitkeep just yet.

Reply:
no, we no longer need to worry about the routes because they are now dynamically handled by TSOA which creates a single file for the openapi along with validators and etc.